### PR TITLE
Added automatic closing of hood after 1s

### PR DESCRIPTION
**Pull request by c0d3rman**
and made delays RobotMap constants

### CODE REVIEW

File: src/org/usfirst/frc4904/robot/RobotMap.java [hunk 1]
```diff
@@ -82,6 +82,8 @@
 			public static final double TURN_EXP = 2;
 			public static final double DEFENSE_MANIPULATOR_SPEED_SCALE = 0.25;
 			public static final double OPERATOR_Y_OUTTAKE_UPPER_THRESHOLD = -0.5;
+			public static final double FLYWHEEL_SPINUP_AFTER_HOODUP_DELAY = 0.25;
+			public static final double HOODDOWN_AFTER_TRIGGERRELEASE_DELAY = 1;
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/RobotMap.java]: Can we have units for these two constants?
`SECONDS_HOODDOWN_AFTER_TRIGGRRELEASE_DELAY`, etc?

Comment (c0d3rman) [inline src/org/usfirst/frc4904/robot/RobotMap.java]: I guess, but all time is done in seconds. Commit if you want to change it.

**Comment by carturn** [comment]
This should definitely not be button1.whenReleased. If it exists, it should be button2.whenReleased.

**Comment by c0d3rman** [comment]
@carturn Why? It seems like they would have roughly the same behavior - one would lower the hood 1 second after releasing the trigger whereas the other would lower it 1 second after releasing the spin-up button. Both would be safe, as a ball will never be caught in the hood while it lowers.

**Comment by carturn** [comment]
I think the expanded flywheel might interfere with the hood. Also, button2 currently raises the hood. The operator would typically be holding button2 for the entire duration of a shooting operation, while button1 is pressed only very briefly. Releasing button2 represents the termination of an entire firing sequence, releasing button1 represents the termination of the ball inject stage.

**Comment by c0d3rman** [comment]
I am less worried about the flywheel, but I agree both on the representation point and on the fact that if button1 then button2 were pressed the hood would hang up. Changing.

**Comment by carturn** [comment]
:+1:

**Pull request merged**